assets/xml/objects/object_link_goron.xml

```diff
@@ -90,7 +90,7 @@
         <Limb Name="gLinkGoronShieldingArmsAndLegsLimb" Type="Standard" EnumName="LINK_GORON_SHIELDING_LIMB_ARMS_AND_LEGS" Offset="0x1779C" />
         <Skeleton Name="gLinkGoronShieldingSkel" Type="Flex" LimbType="Standard" LimbNone="LINK_GORON_SHIELDING_LIMB_NONE" LimbMax="LINK_GORON_SHIELDING_LIMB_MAX" EnumName="LinkGoronShieldingLimb" Offset="0x177B8" />
 
-        <!-- <Blob Name="object_link_goron_Blob_0178D0" Size="0x10" Offset="0x178D0" /> -->
+        <Animation Name="object_link_goron_Anim_0178D0" Offset="0x178D0" />
```

Comment:
Might as well just name it since either it's the shielding one and we know what it is, or it's unused?